.github/workflows/pythonapp.yml

```diff
@@ -76,7 +76,7 @@ jobs:
       if: ${{ matrix.python-version != '3.5' && matrix.os != 'windows-2019' }}
       id: cache
       env:
-        CACHE_NUMBER: 2
+        CACHE_NUMBER: 1
```

Comment:
this is not fixing the "mac is rebuilding TVM over and over again" problem...

Reply:
hmmm nor on ubutntu actually! maybe it is because it's not building `main`? or is this a new issue?

Reply:
In !374 https://github.com/microsoft/hummingbird/pull/374/checks?check_run_id=1541456379 it skips tvm if we check in only the last commit in this PR